duckduckgo/iOS
# hotfix/7.6.1.2 - fix crash and error flash

Task/Issue URL: https://app.asana.com/0/414235014887631/699825463344755
Tech Design URL:
CC:

**Description**:

During testing of 7.6.1.1 discovered a crash when using the fire button and observed an error "flash" (which has been there since before this but seems more obvious now)

**Steps to test this PR**:

Crash:
1. Use the fire button
1. Tap the tabs button
1. Use the fire button
1. Tap the tabs button
1. Use the fire button
1. (repeat a few times)
1. Should be no crash

Error flash
1. Use the fire button
1. Kill the app
1. Launch the app and load a non-DDG bookmark
1. Search for BBC
1. Tap top link
1. Should be no error "flash"
1. Ensure regular errors are handled correctly (e.g. invalid URL, no network, etc)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
